MVC/view/CLI.py

```diff
@@ -76,8 +94,68 @@ def showHoneyComb(self):
                     %s       %s
                         %s      
             ''' % (self.displayLetters[0], self.displayLetters[1],self.displayLetters[2], self.controller.controllerGetReqLetter(), self.displayLetters[3],self.displayLetters[4],self.displayLetters[5]))
+    def newPuzzle(self):
+        if(self.controller.controllerGetPuzzleState() == 1):
+            wantSave = input("Hey do you want to save the game? (yes/no): ")
+            if (wantSave.lower() == "yes"):
+                inputFile = input("Please choose a name for the file: ")
+                print("Saving your game!")
+                self.controller.controllerSaveGame(inputFile)
+            else:
+                print("Ok, lets generate a new puzzle! ")
+        self.controller.controllerNewGame()
+        isAuto = input("Do you want it to be automatically generated? (yes/no): ")
+        while isAuto.lower() != "yes" and isAuto.lower() != "no":
+            isAuto = input("Do you want it to be automatically generated? (yes/no): ")
+    
+        if (isAuto.lower() == "yes") or isAuto.lower() == "no":
+            self.checkGame(isAuto)
+
+    def showPuzzle(self):
+        if (self.controller.controllerGetPuzzleState() == 0):
+            print("No game started!")
+        else:
+            print("Your letters: " + self.controller.controllerGetLetters())
+            print("Required letter: " + self.controller.controllerGetReqLetter())
+            print("Guessed words: " + str(self.controller.controllerGetGuessedWordsCLI()))
+            self.showHoneyComb()
+    def showFoundWords(self):
+        if (self.controller.controllerGetPuzzleState() == 0):
+                    print("No game started!")
+        else:
+            print("Guessed words: " + str(self.controller.controllerGetGuessedWordsCLI()))
+    
+    def shuffleLetters(self):
+        if (self.controller.controllerGetPuzzleState() == 0):
+                print("No game started!")
+        else:
+            self.controller.controllerShuffleAuto()
+            self.showHoneyComb()
+            print("Your letters: " + self.controller.controllerGetLetters())
+    def savePuzzle(self):
```

Comment:
Spacing again here.. Also we need to be using consistent commenting. Using the ''' what the method does blah blah''' format.